.github/workflows/pre-merge-checks.yml

```diff
@@ -24,3 +24,35 @@ jobs:
           cache: gradle
       - name: Build
         run: ./gradlew clean build
+  build-evcs-stub:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Install
+        working-directory: ./di-ipv-evcs-stub/lambdas
+        run: npm ci
+      - name: Build
+        working-directory: ./di-ipv-evcs-stub/lambdas
+        run: npm run build
+      - name: Test
+        working-directory: ./di-ipv-evcs-stub/lambdas
+        run: npm run test
+  build-ticf-stub:
```

Comment:
I'm wondering if it's worth splitting these out - seems like if you raise a PR against the evcs stub, for example, you'll end up building all the java projects too. I'm not sure if/how we pay for GHA compute, and it's probably minimal anyway, but worth considering?

Reply:
So we also have `pre-merge-ticf.yaml` etc., and exclude those paths from this one?

It might be nice, although I think not critical. GHA compute is an issue - there's a free tier limit for the org, but atm this will be dwarfed by the IPV Core integration test runs 😬

Reply:
Yeah. that's what i was thinking. If we go down that path it might be worth splitting apart the java projects too

Reply:
I'm going to merge this as-is, and consider doing this as a future enhancement - the volume of PRs in this repo isn't super high anyway :)